closed/src/java.base/windows/native/libjncrypto/NativeCrypto_md.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2019, 2019 All Rights Reserved
+ * ===========================================================================
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * IBM designates this particular file as subject to the "Classpath" exception
+ * as provided by IBM in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * ===========================================================================
+ */
+
+#include <windows.h>
+
+#include "NativeCrypto_md.h"
+
+/* Load the crypto library (return NULL on error) */
+void * load_crypto_library() {
+    void * result = NULL;
+    const char *libname;
+    const char *oldname = "libeay32.dll";
```

Comment:
Is this, as the name suggests, only for 32-bit Windows?

Reply:
Yes, I think so.

Reply:
If so, the code should not attempt to open `libeay32.dll` on 64-bit Windows.

Reply:
Responded with doing any investigation and it turns out I am wrong. The libeay32.dll is the dll used for OpenSSL 1.0.2 and it seems they use the same filename for both 32 and 64bit.

Reply:
Thanks for following up.